vite.config.ts

```diff
@@ -59,6 +66,22 @@ export default defineConfig({
           ? []
           : ['vee-validate', '@vee-validate/rules']),
       ],
+      input: [
+        path.resolve(__dirname, 'src/index.ts'),
+        path.resolve(__dirname, 'src/icons.ts'),
+        path.resolve(__dirname, 'src/rules/index.ts'),
+        path.resolve(__dirname, 'src/resolvers/index.ts'),
+        path.resolve(__dirname, 'src/components/router/index.ts'),
+      ],
+      output: {
+        preserveModules: true,
+        entryFileNames: ({ name }) => {
+          return `[format]/${name
+            .replace(/^src\//, '')
+            .replace(/\.(vue|svg)$/, '')
+            .replace(/^node_modules/, 'third-party')}.js`;
+        },
```

Comment:
Same comment that on the other PR. And maybe `vendor` here too:
```suggestion
        entryFileNames: ({ name }) => {
          return `[format]/${name
            .replace(/^src\//, '')
            .replace(/\.(vue|svg)$/, '')
            .replace(/^node_modules/, 'vendor')}.js`;
        },
```